### PR TITLE
Update pmm-update to 2.22.0

### PR DESCRIPTION
**Pull request by nikita-b**

### CODE REVIEW

File: rhel/SPECS/pmm-update.spec [hunk 1]
```diff
@@ -76,6 +76,9 @@ install -p -m 0755 bin/pmm-update %{buildroot}%{_sbindir}/
 # Specifically, the change date is ignored – RPM's "Buildtime" is used instead.
 
 %changelog
+* Wed Sep 08 2021 Nikita Beletskii <nikita.beletskii@percona.com> - 2.22.0-62
```
Comment (BupycHuk) [inline rhel/SPECS/pmm-update.spec:79]: @denisok should it be today or release date?

**Review by denisok — APPROVED** [review]

File: rhel/SPECS/pmm-update.spec (hunk 1 above)
Comment (denisok) [reply rhel/SPECS/pmm-update.spec:79]: no, date 21st Sep

Comment (denisok) [reply rhel/SPECS/pmm-update.spec:79]: please change

**Review by denisok — CHANGES_REQUESTED** [review]

File: rhel/SPECS/pmm-update.spec (hunk 1 above)
Comment (denisok) [reply rhel/SPECS/pmm-update.spec:79]: no, incorrect. We actually take the date from Repo, so this date doesn't matter.

**Review by denisok — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**